src/convertToObject.js

```diff
@@ -3,16 +3,42 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example in
+ * [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const arrOfProperties = sourceString.split(';');
+
+  for (const index in arrOfProperties) {
+    let elem = arrOfProperties[index];
```

Comment:
instead of using for...in loop, use for...of

```
 for (const prop of properties) 
 ```
 
 so in `prop` you will get the same `let elem = arrOfProperties[index];`

Reply:
in this case later I have to find index of prop and change the element in arrOfProperties (it is outside of loop)
so I have decided to use for ... in loop instead of for ... of loop. (line 26 in my code)